humidity_sensor/humidity_sensor.ino

```diff
@@ -22,13 +22,8 @@ const int led2 = 12;
 const int led3 = 11;
```

Comment:
Unify constants to match naming conventions and specify they refer to Pin numbers.

E.g. cons int LED1_PIN = 13;
BUTTON_PIN = 7;